spec/implementations/pto_next_week/fetch_next_week_pto_from_drive_spec.rb

```
@@ -0,0 +1,62 @@
+    allow(mocked_shared_storage_writer).to receive(:write).and_return(
+      [{ 'status' => 'success', 'id' => 1 }]
+    )
+
+    allow(mocked_shared_storage_writer).to receive(:set_processed).and_return(nil)
+    allow(mocked_shared_storage_writer).to receive(:update_stage).and_return(true)
+    allow(mocked_shared_storage_writer).to receive(:set_in_process).and_return(nil)
+
+    allow(mocked_shared_storage_reader).to receive(:set_processed).and_return(nil)
+    allow(mocked_shared_storage_reader).to receive(:set_in_process).and_return(nil)
+
+    @bot = Implementation::FetchNextWeekPtosFromGoogleSheets.new(options, mocked_shared_storage_reader, mocked_shared_storage_writer)
```

RuboCop (1.75.5)

[convention] Line is too long. [133/120]

(Layout/LineLength)